Dependencies/AndroidExtensions/Include/AndroidExtensions/JavaWrappers.h

```diff
@@ -259,6 +261,18 @@ namespace android::content::res
     };
 }
 
+namespace android::graphics
+{
+    class SurfaceTexture : public java::lang::Object
+    {
+    public:
+        SurfaceTexture();
+
+        void initWithTexture(int texture);
+        void updateTexture() const;
```

Comment:
This should match the Java API naming:
```suggestion
        void updateTextImage() const;
```